Samples/Tutorial Samples/WinMLPIXSample/WinMLPIXSample/WinMLHelper.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+#include "pch.h"
+
+#ifndef WINMLHELPER_H
+#define WINMLHELPER_H
+
+using namespace winrt;
+using namespace winrt::Windows::AI::MachineLearning;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Foundation::Collections;
+using namespace winrt::Windows::Graphics::Imaging;
+using namespace winrt::Windows::Media;
+using namespace winrt::Windows::Storage;
+using namespace std;
+
+// Global variables
+static hstring modelPath = L".\\Assets\\SqueezeNet.onnx";

```

Comment:
I think we have a different code convention for globals, but I can't remember what it is....

Reply:
You mean naming conventions? seems camel casing IS the conversion from many other projects...

Reply:
yes, but globals have a g_ or are all caps, or something like that.